features/step_definitions/buyer_steps.rb

```diff
@@ -18,173 +21,163 @@
   buyer.buy! provider.account_plans.default
 end
 
-Given /^a buyer "([^"]*)" of (provider "[^"]*")$/ do |org_name, provider|
-  account = FactoryBot.create(:buyer_account, :provider_account => provider,
-                    :org_name => org_name)
+Given "a buyer {string} of {provider}" do |org_name, provider|
```

Comment:
I would prepend `there is` to the step expression, but given old step is like that, you can ignore.

Reply:
I see there is an existing convention so it should be fine.